src/main/java/frc/robot/subsystems/TankSubsystem.java

```diff
@@ -20,37 +23,57 @@ public class TankSubsystem extends SubsystemBase {
   private final WPI_TalonSRX rightMotor;
   private final WPI_TalonSRX rightMotorFollower;
 
+  private final XboxController controller;
+
+  private double leftSpeed;
+  private double rightSpeed;
+  
   /** Creates a new TankSubsystem. */
   public TankSubsystem() {
     /* Left Motors */
     leftMotor = new WPI_TalonSRX(LEFT_FRONT_MOTOR_ID);
     leftMotorFollower = new WPI_TalonSRX(LEFT_BACK_MOTOR_ID);
     leftMotor.setNeutralMode(NeutralMode.Brake);
+    leftMotorFollower.setNeutralMode(NeutralMode.Brake);
     leftMotorFollower.follow(leftMotor);
 
     /* Right Motors */
     rightMotor = new WPI_TalonSRX(RIGHT_FRONT_MOTOR_ID);
     rightMotorFollower = new WPI_TalonSRX(RIGHT_BACK_MOTOR_ID);
     rightMotor.setNeutralMode(NeutralMode.Brake);
+    rightMotorFollower.setNeutralMode(NeutralMode.Brake);
     rightMotor.setInverted(InvertType.InvertMotorOutput);
     rightMotorFollower.setInverted(InvertType.FollowMaster);
     rightMotorFollower.follow(rightMotor);
+
+    /* Controls */
+    controller = new XboxController(OperatorConstants.kDriverControllerPort);
```

Comment:
@CrolineCrois @rishay-jain Please confirm, but I think this should go in the constructor of RobotContainer, and then you can call arcadeDrive from the RobotContainer periodic. This makes it so we can use the other buttons on the controller for other functions.

Reply:
@MysticalApple I think you mentioned this in class and we might have given you the wrong answer. We probably won't give the drive controller any more buttons during shop project, but I think its good practice to put all button bindings in RobotContainer so they can be easily accessed.